app/scripts/controllers/groups/EditGroupController.js

```diff
@@ -107,6 +107,10 @@
                     externalId: data.externalId,
                     staffId: data.staffId
                 };
+                if (data.timeline.submittedOnDate) {
+                    const submitDate = dateFilter(data.timeline.submittedOnDate, scope.df);
```

Comment:
instead you can write the code inside 'resourceFactory.groupResource.get  call'  or you can use the existing variable.